internal/backend/temporalclient/config.go

```diff
@@ -51,6 +53,12 @@ var (
 	Configs = configset.Set[Config]{
 		Default: &Config{
 			Monitor: defaultMonitorConfig,
+			DataConverter: DataConverterConfig{
+				Compress: true,
+				Encryption: DataConverterEncryptionConfig{
+					Encrypt: true,
```

Comment:
will this work with encrypt true but no keys provided by default ?

Reply:
disagree here - i think this will require the user to specify keys when used in production. if you don't set these - this will fail and let the user know that keys are missing.